google_photos_takeout_helper/__main__.py

```diff
@@ -233,34 +251,27 @@ def find_duplicates(path: Path, filter_fun=lambda file: True):
 
                 files_by_full_hash[full_hash].append(file)
 
-        # Now we have the final multimap of absolute dups, We now can attempt to find the original file
+    # Removes all duplicates in folder
+    def remove_duplicates(dir: Path):
+        find_duplicates(dir, lambda f: (is_photo(f) or is_video(f)))
+        nonlocal s_removed_duplicates_count
+
+        # Now we have populated the final multimap of absolute dups, We now can attempt to find the original file
         # and remove all the other duplicates
         for files in files_by_full_hash.values():
             if len(files) < 2:
                 continue # this file size is unique, no need to spend cpu cycles on it
-            original = None
-            for file in files:
-                if not _re.search(r'\(\d+\).', file.name):
-                    original = file
-            if original is None:
-                original = files[0]
 
-            dups = files.copy()
-            dups.remove(original)
-            duplicates += dups
-
-        return duplicates
-
-    # Removes all duplicates in folder
-    def remove_duplicates(dir: Path):
-        duplicates = find_duplicates(dir, lambda f: (is_photo(f) or is_video(f)))
-        for file in duplicates:
-            file.unlink()
-        nonlocal s_removed_duplicates_count
-        s_removed_duplicates_count += len(duplicates)
+            s_removed_duplicates_count += len(files) - 1
+            for file in files:
+                #TODO reconsider now that we're searching globally
+                #check which duplicate has best exif?
```

Comment:
Um
```python
for files in files_by_full_hash
```

If two files have *identical full hash*, they also have *identical* exif - do they?

Reply:
That's correct, I'll remove this comment.

Reply:
I was trying to make an edge case where there was none 🤪.

Reply:
We can probably still
>  select ones that have corresponding json

Just select one with shortest name (so no `(1)` stuff)